components/GroupSearchFilters/GroupSearchFilters.js

```diff
@@ -0,0 +1,104 @@
+import PropTypes from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
+import { useModalDispatch, showModal } from 'providers/ModalProvider';
+import { useGroupFilters, resetValues } from 'providers/GroupFiltersProvider';
+import { Box, Button } from 'ui-kit';
+
+function CurrentFiltersList(props = {}) {
```

Comment:
General callout that this `GroupSearchFilters` file is very prototype-y, in the sense that I just needed a closet to shove some UI functionality into in lieu of super clear direction on next steps.

The purpose of this component — which admittedly isn't clear from the name — is to show and allow modification of the Group Search filters on the search results page.

For now, it just opens the modal flow again, but we'll circle back on this.